src/subscription_manager/managercli.py

```diff
@@ -89,7 +89,7 @@ def __init__(self):
 
     def main(self) -> Optional[int]:
```

Comment:
as a consequence of the change to `CLI.main()` from `Optional[int]` to `int`, this needs to be changed to `int` as well, as the only `return` here is with the return value of `CLI.main()`